dynamic_rest_client/query.py

```diff
@@ -167,7 +167,7 @@ def __iter__(self):
                 if self._page == self._pages:
                     # end of results
                     self._reset()
-                    raise StopIteration()
+                    return
```

Comment:
Necessary for new 3.7 behavior

> PEP 479 is enabled for all code in Python 3.7, meaning that StopIteration exceptions raised directly or indirectly in coroutines and generators are transformed into RuntimeError exceptions. (Contributed by Yury Selivanov in bpo-32670.)

https://stackoverflow.com/questions/51700960/runtimeerror-generator-raised-stopiteration-every-time-i-try-to-run-app

Reply:
interesting, should be careful to test vishnu-backend on a feature branch when updating this repo there